doc/sphinx/source/community/dataset.rst

```diff
@@ -42,14 +42,15 @@ and run the recipe, to make sure the CMOR checks pass without warnings or errors
 
 To test a pull request for a new CMORizer script:
 
-#. Download the data following the instructions included in the script and place
-   it in the ``RAWOBS`` path specified in your ``config-user.yml``
+#. Download the data following the instructions included in the script and
+   place it in the ``RAWOBS`` `rootpath` specified in your :ref:`configuration
```

Comment:
Use double backticks around `rootpath` to make it monospace (a single backtick is italic and the convention is to use monospace for things that have to be typed exactly like that)

Reply:
Changed in https://github.com/ESMValGroup/ESMValTool/pull/3761/commits/ce62d9f57c8b141f831f0c1f61738752b9f4e6c1 and https://github.com/ESMValGroup/ESMValTool/pull/3761/commits/75e2b3363d25f54602c65767a56cfc035a7630eb

Reply:
Thanks! I assumed our convention was to use monospace font for things that need to be written exactly like that to work and all the other cases were people who just forgot that in .rst you need double backticks to create monospace fonts. Maybe we need to write this in our contributor docs somewhere too. From the text you linked it looks like numpy plans to adopt the same convention, but are working on some feature to make that work with single backticks:

> This guide continues to recommended that parameter names be enclosed within single backticks. Currently, this may cause parameter names to render improperly and cause warnings, but numpydoc will soon release a feature that causes them to render as monospaced hyperlinks to the parameter documentation.